Introduce mix_blend

sashkent3 opened the pull request:
This PR introduces support for [Spine's MixBlend](https://en.esotericsoftware.com/spine-api-reference#TrackEntry-mixBlend).
Closes #222

AGulev reviewed (approved):
Looks great!
Thank you!

sashkent3 commented:
> @sashkent3 could you please also add an example scene with usage of this feature? (it is useful as example for people wanna use it, but also for tests)

@AGulev I've added an example on using `spine.MIX_BLEND_ADD` based on [the Spine's owl example](https://en.esotericsoftware.com/spine-examples-owl), modified to conform with the below.
I would like to note that there's a very important caveat about working with the additive mix blend mode ([ref1](https://esotericsoftware.com/spine-api-reference#MixBlend-add), [ref2](https://esotericsoftware.com/forum/d/28796-mixblend-add-bugging-on-owl-example-project/2)). We should probably mention this somewhere to avoid confusion. I'm not sure what would be a good place for the warning.

AGulev commented:
> I would like to note that there's a very important caveat about working with the additive mix blend mode ([ref1](https://esotericsoftware.com/spine-api-reference#MixBlend-add), [ref2](https://esotericsoftware.com/forum/d/28796-mixblend-add-bugging-on-owl-example-project/2)). We should probably mention this somewhere to avoid confusion. I'm not sure what would be a good place for the warning.

It is possible to add it in documentation:
https://github.com/defold/extension-spine/blob/main/docs/index.md

sashkent3 commented:
> > I would like to note that there's a very important caveat about working with the additive mix blend mode ([ref1](https://esotericsoftware.com/spine-api-reference#MixBlend-add), [ref2](https://esotericsoftware.com/forum/d/28796-mixblend-add-bugging-on-owl-example-project/2)). We should probably mention this somewhere to avoid confusion. I'm not sure what would be a good place for the warning.
> 
> It is possible to add it in documentation: https://github.com/defold/extension-spine/blob/main/docs/index.md

I've added a small section to the docs with relevant references.

AGulev commented:
@sashkent3 perfect! thank you!
